app/routes/_dashboard.blogposts.tsx

```diff
@@ -26,9 +24,9 @@ const Blogposts = () => {
         />
       )}
 
-      {otherBlogposts.length > 0 && (
+      {blogposts.length > 0 && (
         <CardWithMultipleContent
-          content={otherBlogposts}
+          content={blogposts}
```

Comment:
hmm. Vil de som er featured dukke opp to ganger nå? En gang i featured og en gang i "alle bloggposter" ?

Reply:
kanskje vi burde sjekke iden til de som er i "featuredBlogposts" opp mot alle blogposts slik at vi ikke viser de dobbelt?

Reply:
samme for lectures og videos

Reply:
dette burde strengt tatt vært i en egen branch/PR da men 🙈

Reply:
Jo, de vil dukke opp en gang i hver (ble tatt opp på slack). Men enig i at det kan være et eget PR - will fix.